R/ard_attributes.R

```diff
@@ -1,29 +1,48 @@
-#' ARD Attributes
+#' Add variable attributes to an ARD object
```

Comment:
I think we can keep the title simple in this case

```suggestion
#' ARD Attributes
```